package.json

```diff
@@ -5,13 +5,12 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1",
-    "sinon": "^7.4.1"
+    "react-scripts": "3.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3000 react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --coverage --watchAll=false --detectOpenHandles",
```

Comment:
So I understand:
This prints the coverage, it disables the watcher and the final part is for Travis?

Reply:
yes